src/Pages/Aplication/Aplication.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './Aplication.scss';
+import { useState } from "react";
+import {Link} from 'react-router-dom'
+
+const Aplication = () => {
+
+    const [isCheckedFirst, setIsCheckedFirst] = useState (false);
+    const [isCheckedSecond, setIsCheckedSecond] = useState (false);
+
+    const handleOnChange = (something, someState) => {
+        something(!someState)
+    }
+
+    return(
+        <div className="aplication">
+            <div className='aplicationDescription'>
+                <h3 className='aplicationHeader'> Privacy Notice & Terms and Conditions agreement</h3>
+                <p className='aplicatonText'>You are about to start the application process for EY-NASA mission to Mars. Please read <span> </span>
+                    <Link to="/privacy" className="privacy">„Privacy Notice“</Link> <span> </span> and <span> </span>
+                    <Link to="/terms" className="terms">„Terms and Conditions“</Link> <span> </span>
+                        before giving your consent. If you agree, EY and NASA will use the data for the purposes of the 
+                        legitimate interest(s) of the Mars mission.The specific legitimate interest(s) are the 
+                        provision of mission planning and are for internal use by EY and NASA employees.</p>
+            </div>
+            <div className='aplicationProceed'>
+                <div className='aplicationCheckbox'>
+                    <div>
+                        <input type="checkbox" id='readDocumentations' name='readDecumentations'
+                            checked={isCheckedFirst} onChange={()=>handleOnChange(setIsCheckedFirst, isCheckedFirst)}/>
```

Comment:
Using anonymous functions in render is a bad practice since on every render a new instance of function would be created and that can cause memory issues, but since you are passing parameters here try to implement it different 
[React docs for functions callbacks](https://reactjs.org/docs/faq-functions.html)

Reply:
im not sure how to implement this other way. i used callback because i need to send 2 parameters to the functions. i read this article you have sent me, but not sure what else i could do.